### DISCUSSION TITLE
Have you ever consider to integrate a FileSystem inside MCUBOOT ?

### DISCUSSION THREAD

**Discussion by Gioelinho** [mcu-tools/mcuboot#1418] [Ideas]
Have you ever consider to integrate a FileSystem inside MCUBOOT like LFS, EmFile or something else?

This could help you to save old/new firmware or organize it in folders.

**Comment by de-nordic** [comment]
Your application should be ding that and loading desired image to secondary slot and marking it for next boot.
Adding support for filesystems in MCUboot would bring a lot of problems and security issues.
